test/include.test.js

```diff
@@ -579,6 +595,15 @@ describe('include', function() {
           });
         });
 
+        it('does not return included item if hasOne is missing the id property', function(done) {
+          User.findOne({include: {relation: 'profile'}, fields: 'name'}, function(err, user) {
+            should.not.exist(err);
```

Comment:
```js
if (err) return done(err);
```

"should not exist" check discards the stack trace from the original error, making the test failure difficult to troubleshoot.